docs/can-guides/experiment/crud-beginner/crud-beginner.md

```diff
@@ -424,16 +430,16 @@ also [can-stache-bindings.event#on_VIEW_MODEL_OR_DOM_EVENT__KEY_VALUE_ set prope
 Let’s examine this part of the code:
 
 @sourceref ./8.js
-@highlight 29-31,40,only
+@highlight 29-31,43,only
 
-`on:click="this.selected = todo"` will cause the ViewModel’s `selected` property to be set
+`on:click="this.selected = todo"` will cause the custom element’s `selected` property to be set
 to the `todo` when the `<span>` is clicked.
 
 The template above also changes the span’s class depending on [can-stache.helpers.if if]
 `todo.complete` is true or false.
 
 Additionally, we add [can-define.types.propDefinition#function__ `selected: Todo`]
-to the ViewModel. In our app, we only ever set `selected` to an instance of a `Todo`,
+to the custom element. In our app, we only ever set `selected` to an instance of a `Todo`,
```

Comment:
This will type check also, `type.convert(Todo)` will convert plain objects. Since this app only ever sets it to todos, we should just explain that this would throw.

Reply:
I think I’ll change this to:

> In our app, we only ever set `selected` to an instance of a `Todo`, otherwise CanJS would throw an error. If we wanted plain objects to be converted into new Todo instances, we could use [type.convert(Todo)](https://canjs.com/doc/can-type/convert.html).